tests/Unit/Evolution/Actions/Test_ComputeVolumeDuDt.cpp

```diff
@@ -0,0 +1,61 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "DataStructures/DataBox.hpp"
+#include "DataStructures/DataBoxTag.hpp"
+#include "Domain/ElementId.hpp"
+#include "Domain/ElementIndex.hpp"
+#include "Evolution/Actions/ComputeVolumeDuDt.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+#include "tests/Unit/ActionTesting.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+namespace {
+struct var_tag : db::DataBoxTag {
+  using type = int;
+  static constexpr db::DataBoxString label = "var_tag";
+};
+struct dt_var_tag : db::DataBoxTag {
+  using type = int;
+  static constexpr db::DataBoxString label = "dt_var_tag";
+};
+
+struct ComputeDuDt {
+  using return_tags = tmpl::list<dt_var_tag>;
+  using argument_tags = tmpl::list<var_tag>;
+  static void apply(const gsl::not_null<int*> dt_var, const int& var) {
+    *dt_var = var * 2;
+  }
+};
+
+struct System {
+  using du_dt = ComputeDuDt;
+};
+
+using ElementIndexType = ElementIndex<2>;
+
+struct Metavariables;
+using component = ActionTesting::MockArrayComponent<
+    Metavariables, ElementIndexType, tmpl::list<>,
+    tmpl::list<Actions::ComputeVolumeDuDt<2>>>;
+
+struct Metavariables {
+  using component_list = tmpl::list<component>;
+  using system = System;
+};
+}  // namespace
+
+SPECTRE_TEST_CASE("Unit.Evolution.dG.ComputeVolumeDuDt",
```

Comment:
dG?

Reply:
derp, I used to have it in the `namespace dg`, will remove and  rebase this on top of #474